### PR TITLE
Windows CMake installer improvements

### PR DESCRIPTION
**Pull request by JosepMaJAZ**
Corrections to the doc installation path (CMake)
Inclusion of the current QT 5 translation files (CMake and Scons)
Corrected one resource path in glslwaveformrenderersignal.cpp
Cleaned mixxx.qrc

- [x] Path fixes (CMake)
- [x] Inclusion of new qt translations (CMake, Scons)
- [ ] Installer translations (CMake)
- [ ] runtime installation (CMake)

It seems that for installer translations,  the loop and regeneration implemented in the Scons script is needed. In other words, to generate a multi-language installer, first one needs to create N localized installers, extract the "difference" and generate a new installer adding those templates.

### CODE REVIEW

File: src/waveform/renderers/glslwaveformrenderersignal.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@ bool GLSLWaveformRendererSignal::loadShaders() {
     m_frameShaderProgram->removeAllShaders();
 
     if (!m_frameShaderProgram->addShaderFromSourceFile(
-            QGLShader::Vertex, ":shaders/passthrough.vert")) {
+            QGLShader::Vertex, ":/shaders/passthrough.vert")) {
```
Comment (Be-ing) [inline src/waveform/renderers/glslwaveformrenderersignal.cpp:48]: Maybe this was the cause of https://bugs.launchpad.net/mixxx/+bug/1872172 ?

Comment (JosepMaJAZ) [reply src/waveform/renderers/glslwaveformrenderersignal.cpp:48]: Not sure, but this only affects the glsl one (although I believe currently that's the default renderer in new setups)

**Comment by Be-ing** [comment]
Reassigning to 2.3 so we can use AppVeyor for official builds instead of our unreliable build sever.

**Comment by Be-ing** [comment]
AppVeyor Windows build succeeded. No need to wait for the rest.

**Pull request merged**